content/en/users/workload-manager/_index.md

```diff
@@ -4,9 +4,460 @@ linkTitle: "Workload Manager"
+    * [Send your certificate's subject to the DIRAC team](#send-your-certificates-subject-to-the-dirac-team)
+  * [The EGI Workload Manager Web Portal](#the-egi-workload-manager-web-portal)
+    * [Upload Proxy](#upload-proxy)
+    * [Job Submission](#job-submission)
+    * [Monitor Job status](#monitor-job-status)
+    * [Get Results from Sandbox](#get-results-from-sandbox)
+    * [Full User Guide for DIRAC Web Portal](#full-user-guide-for-dirac-web-portal)
+  * [The DIRAC client tool](#the-dira-client-tool)
+    * [Managing simple jobs](#managing-simple-jobs)
+    * [Jobs with Input Sandbox and Output Sandbox](#jobs-with-input-sandbox-and-output-sandbox)
+    * [More details](#more-details)
+    * [Technical Support](#technical-support)
+
+## What is the EGI Workload Manager?
+
+The EGI Workload Manager is a service provided to the EGI community to
+efficiently manage and distribute computing workloads on the EGI infrastructure.
+The service, based on the DIRAC technology, is configured to support a number
+of HTC and cloud resource pools from the EGI Federation.
+This pool of computing resources can be easily extended and customized to
+support the needs of new scientific communities.
+In the LHCb experiment the service has proven production scalability up to peaks
+of more than 100 thousand concurrently running jobs.
+WeNMR, the structural biology community, uses the service for a number of
+community services.
+The community reported an improvement of jobs submission from previous 70% to
+99% with the EGI Workload Manager service.
+The delivery of the service is coordinated by the EGI Foundation and operated by
+IN2P3 on resources provided by CYFRONET.
+
+## Main Features
+
+The EGI Workload Manager:
+Maximizes usage efficiency by choosing appropriately computing and storage
+resources on real-time.
+Provides a large-scale distributed environment to manage and handle data
+storage, datamovement, accessing and processing.
+Handles job submission and workload distribution in a transparent way.
+Improves the general job throughput compared with native management of EGI
+grid or cloud computing resources.
+Offers pilot-based task scheduling method, that submits pilot jobs to
+resources to check the execution environment before to start the user's jobs.
+From a technical standpoint, the user's job description is delivered to the
+pilot, which prepares its execution environment and executes the user application.
+The pilot-based scheduling feature solves many problems of using
+heterogeneity and unstable distributed computing resources.
+Includes easy extensions to customize the environment checks to address the
+needs of a particular community.
+Users can choose appropriately computing and storage resources maximising their
+usage efficiency for particular user requirements.
+Handles different storage supporting both cloud and grid capacity.
+Provides a user-friendly interface that allows users to choose among different
+DIRAC services.
```

Comment:
Should this be a list?

Reply:
09f1d0a52bc9c084062f9f05a5ae58e07f898200